src/plugins/drawing-tool/components/drawing-layer.tsx

```diff
@@ -189,6 +193,18 @@ export class DrawingLayerView extends React.Component<DrawingLayerViewProps, Dra
     let moved = false;
     const {selectedObjects, hoverObject } = this.state;
     let { objectsBeingDragged } = this.state;
+
+    // Prevents dragging multiple objects when you have one (or more) selected
+    // and then drag one unless shift/cmd pressed.
```

Comment:
This is more nuanced. If the object you are dragging is selected then the selection should not be cleared and all objects should be moved. If the object you are dragging was not selected then only then all of the other objects should be deselected and just the object you are dragging should be selected.